frontend/public/style/_overrides.scss

```diff
@@ -370,8 +374,12 @@ button.pf-v5-c-dropdown__menu-item.pf-m-disabled {
   pointer-events: auto;
 }
 
-#modal-container .pf-v5-c-backdrop {
-  position: absolute !important;
+#modal-container {
+  // PatternFly sets `overflow: hidden` which is problematic when using appendTo in <Modal>
```

Comment:
I've identified an alternate fix that doesn't require an override, but will wait for the current CI test run to finish before pushing changes.